src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -134,54 +134,50 @@ public void should_attach_exception_record_to_the_existing_case_with_evidence_do
     }
 
     @Test
-    public void should_attach_supplementary_evidence_with_pending_payments() throws Exception {
-        //given
-        CaseDetails caseDetails = ccdCaseCreator.createCase(emptyList(), now()); // with no scanned documents
-        CaseDetails exceptionRecord = createExceptionRecord(
-            "envelopes/supplementary-evidence-envelope-with-payment.json"
+    public void should_attach_exception_record_with_pending_payments_when_classification_allows() throws Exception {
+        verifyExceptionRecordWithPendingPaymentsAttachesToCase(
+            "envelopes/supplementary-evidence-envelope-with-payment.json",
+            true
         );
-
-        // when
-        attachExceptionRecord(caseDetails, exceptionRecord, null);
-
-        //then
-        await("Exception record with pending payments is attached to the case")
-            .atMost(60, TimeUnit.SECONDS)
-            .pollDelay(2, TimeUnit.SECONDS)
-            .until(() -> isExceptionRecordAttachedToTheCase(caseDetails, 1));
-
-        verifyExistingCaseIsUpdatedWithExceptionRecordData(caseDetails, exceptionRecord, 1);
     }
 
     @Disabled("Functionality not implemented yet")
     @Test
-    public void should_return_error_when_attaching_exception_with_pending_payments() throws Exception {
-        //given
-        CaseDetails caseDetails =
-            ccdCaseCreator.createCase(
-                singletonList(
-                    new Document(
-                        "certificate1.pdf", "154565768", "other", null, Instant.now(), documentUuid, Instant.now()
-                    )),
-                Instant.now()
-            ); // with 1 scanned document
-
-        CaseDetails exceptionRecord = createExceptionRecord(
-            "envelopes/exception-classification-envelope-with-payment.json"
+    public void should_not_attach_exception_record_with_pending_payments_when_classification_is_not_allowed()
+        throws Exception {
+        verifyExceptionRecordWithPendingPaymentsAttachesToCase(
+            "envelopes/exception-classification-envelope-with-payment.json",
+            false
         );
+    }
+
+    private void verifyExceptionRecordWithPendingPaymentsAttachesToCase(
```

Comment:
Because this method verifies that the exception record either attaches or not, maybe it would be good to change it to `verifyIfExceptionRecordWithPendingPaymentsAttachesToCase`.

Reply:
renamed method name